cmd/translate/main_grpc_integration_test.go

```diff
@@ -446,6 +446,10 @@ func randMessages(t *testing.T, override *translatev1.Messages) *translatev1.Mes
 			Fuzzy:       gofakeit.Bool(),
 		}
 
+		if gofakeit.Bool() {
+			gofakeit.Slice(message.Positions)
+		}
```

Comment:
```suggestion
		gofakeit.Slice(message.Positions)
```

Reply:
that's there on purpose, create nil value